pkg/sql/opt/exec/execbuilder/testdata/select_index

```diff
@@ -1336,3 +1336,105 @@ render           ·         ·           (w)     ·
       │          spans     /1-/10      ·       ·
       └── scan   ·         ·           (v, w)  ·
 ·                table     t3@primary  ·       ·
+
+# ------------------------------------------------------------------------------
+# These tests are for the point lookup optimization, which applies to SELECTs
+# and UPDATEs that only need to scan a subset of the column families.
+# ------------------------------------------------------------------------------
+statement ok
+CREATE TABLE t4 (
+  a INT,
+  b INT,
+  c INT,
+  d INT,
+  PRIMARY KEY (a, b),
+  FAMILY (a, b),
+  FAMILY (c),
+  FAMILY (d)
+)
+
+statement ok
+INSERT INTO t4 VALUES (10, 20, 30, 40)
+
+# Point lookup on c does not touch the d family.
+query TTT
+EXPLAIN SELECT c FROM t4 WHERE a = 10 and b = 20
+----
+render     ·      ·
+ └── scan  ·      ·
+·          table  t4@primary
+·          spans  /10/20/0-/10/20/1/2
```

Comment:
Can you remind me why is the end key `/1/2/` but the start key `/0/` again?

Reply:
Yeah, this is the primary column family and column family `1` coalesced together. (They're different lengths because primary CF keys omit the encoding length.)